internal/controllers/s3userclaim/provisioner.go

```diff
@@ -311,3 +342,73 @@ func (r *Reconciler) assembleCephUserSecret(userName, secretName string) (*corev
 
 	return secret, nil
 }
+
+func (r *Reconciler) generateSubUserAccessMap(desiredSubUsers []string,
+	currentSubUsers []admin.SubuserSpec) map[string]string {
+	// Create a hashmap to move all spec and ceph subUsers to it
+	subUserFullIdAccessMap := make(map[string]string)
```

Comment:
I have the same criticism about the naming. `Map` suffix is not required.

Also, less important about the comment, we use the general `map` term instead of hashmap in Golang (remembers about 😃 )

Reply:
Oh, python habits :))
You're right. I'll fix it.

Reply:
Done